src/Roslyn.Suppressions/Roslyn.Suppressions.UnitTests/TestHelper.cs

```diff
@@ -37,11 +37,13 @@ internal static class TestHelper
                 RoslynIssueLine = line
             };
 
-    public static SonarQubeIssue CreateSonarQubeIssue(string ruleId = "any",
-            int? line = null,
-            string filePath = "filePath",
-            string hash = "hash") =>
-            new SonarQubeIssue(
+        public static SonarQubeIssue CreateSonarQubeIssue(string ruleId = "any",
+                int? line = null,
+                string filePath = "filePath",
+                string hash = "hash",
+                bool isSuppressed = true)
```

Comment:
Made the default for isSuppressed true so it does not interfere with existing tests.